app/models/subscription.rb

```diff
@@ -688,6 +688,10 @@ def resubscribe!
     end
   end
 
+  def update_business_vat_id!(vat_id)
+    update!(business_vat_id: vat_id) if vat_id.present? && business_vat_id.blank?
+  end
```

Comment:
@michellelarney should we allow the buyer to use different VAT IDs in the course of a subscription? Or is it like, once they set a certain VAT in a subscription, we shouldn't allow to change that?